ui/contexts/snaps/snap-interface.tsx

```diff
@@ -152,9 +166,9 @@ export const SnapInterfaceContextProvider: FunctionComponent<
   };
 
   const handleInputChangeDebounced = debounce(
-    (name, value) =>
+    (event: UserInputEventType, name: string | undefined, value: unknown) =>
```

Comment:
Why is this parameterized now? This is used exclusively for `InputChangeEvent`

Reply:
`debounce` doesn't infer function types. I was using this before, but it's no longer necessary so I've reverted it.